index.html

```diff
@@ -13,12 +13,80 @@
         <script>
             var c = document.getElementById("CanvaSN1");
             var ctx = c.getContext("2d");
-            ctx.fillStyle = "#FF0000";
-            ctx.fillRect(100,100,20,20);
-            ctx.fillStyle = "#000000";
-            ctx.fillRect(0,0,20,20);
-            ctx.fillStyle = "#e8f8f5";
-            ctx.fillRect(0,0,400,400);
+            var movimientoX = 0;
+            var movimientoY = 0; 
```

Comment:
Puedes en vez de usar dos variables diferentes lo siguiente:
```suggestion
            var movimiento = {x: 1 , y:2};
            movimiento.x = 4; // <- forma de usarlo
            movimiento.y = 3;
            console.log(movimiento.x , movimiento.y); 
```

- [ ] cambiar